requirements.txt

```diff
@@ -40,7 +40,7 @@ qrcode==6.1
 reportlab==3.5.13
 requests==2.21.0
 zeep==3.2.0
-vatnumber==1.2
+python-stdnum==1.10
```

Comment:
Any chance stdnum 1.8 is enough as a base dep? 1.10 is available in Debian 10 LTS, but Ubuntu will only have it in April 2020 with 20.04 LTS. We can always upgrade the dep to 1.10 after April 2020.

From what I can see in the [changelog](https://github.com/arthurdejong/python-stdnum/blob/master/NEWS), the main VAT-related change we have would be the Venezuela VAT number (RIF) which we could perhaps implement ourselves until then?

Reply:
@odony @qdp-odoo 
-> Changed to 1.8
I checked all countries vat number checks in vatnumber 1.2, stdnum 1.8 & odoo (this PR)
To preserve the previous behavior I :
- left the methods for countries already existing in Odoo (maybe check later if pertinent to remove them and use stdnum methods instead when they exists). Those override stdnum.
- copied methods for countries supported in vatnumber but not in stdnum 1.8 (for co, ua & ru) in odoo

Venezuela was not supported before, so I left it that way.